libs/ledgerjs/packages/types-cryptoassets/src/index.ts

```diff
@@ -76,7 +202,7 @@ export type ExplorerView = {
 export type CryptoCurrency = CurrencyCommon & {
   type: "CryptoCurrency";
   // unique internal id of a crypto currency
-  id: string;
+  id: CryptoCurrencyId | "LBRY" | "groestcoin" | "osmo";
```

Comment:
Doesn't seem clean to have to define exception values here. What's the reason for not having them in `CryptoCurrencyId`?

Reply:
It's because `cryptocurrenciesById` is of type `Record<CryptoCurrencyId, CryptoCurrency>`, and for most cryptocurrencies the key is the same as the `id` expect for these 3 exceptions.
So in the object we have

```ts
{
  lbry: {
      id: "LBRY",
      ...
  },
  groestlcoin: {
      id: "groestcoin", // maybe a typo we didn't notice
  },
  osmosis: {
      id: "osmo",
      ...
  }
} 
```
As we don't want to change currency ids I added this union
For future integration we'd only have to add the id in `CryptoCurrencyId`. this will avoid typos like for `groestlcoin` for exemple